bin/proxy.js

```diff
@@ -0,0 +1,45 @@
+require('babel-register');
+
+const http = require('http');
+
+const bunyan = require('bunyan');
+const config = require('config');
+const httpProxy = require('http-proxy');
+
+const log = bunyan.createLogger({
+  name: 'proxy',
+  app: config.get('appName'),
+  serializers: bunyan.stdSerializers,
+});
+
+const proxy = httpProxy.createProxyServer();
+const apiHost = config.get('proxyApiHost', null) || config.get('apiHost');
```

Comment:
So I wasn't entirely sure what the problem was but I looked into it some more.

There was an issue with cookies (these needed to be rewritten by the proxy). I set that up so that the proxy will strip the `Domain` and `Secure` portions of the cookie and you can log in on the frontend and interact with the site (woo). Leaving your first review works, but it doesn't seem to get loaded and you can't update it (this might not be proxy related).

If you try to use the site on a non-mobile browser then it looks like it works but you can't log in since you'll get redirected to `https://addons-dev.allizom.org` by FxA. We could work around this but we'd need to do something in addons-server.

So for now it looks like defaulting to `http://olympia.dev` seems like a good default and using -dev will sort of work if you want. Well enough that we probably don't need to prevent it, anyway.